Paws4Me/Paws4Me/Data/LocalPetNameViewController.swift

```diff
@@ -8,43 +8,59 @@
 import CoreData
 import UIKit
 
-class LocalPetViewController: UIViewController {
+class LocalPetViewController: UIViewController, UITableViewDelegate {
     // MARK: - IBOutlets
     @IBOutlet weak var petNameTableView: UITableView!
+    let context = (UIApplication.shared.delegate as? AppDelegate)?.persistentContainer.viewContext
```

Comment:
private

Reply:
I really need to remember to make these private. Thank you for the reminder. 👍🏻